exercises/01-composite-ui/after/Divergent.Sales.Data/Repositories/OrderRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Divergent.Sales.Data.Models;
+
+namespace Divergent.Sales.Data.Repositories
+{
+    public class OrderRepository : IOrderRepository
+    {
+        public async Task<List<Order>> Orders()
+        {
+            return OrderSeedData.Orders();
```

Comment:
We abstracted away OrderSeedData in a repository that's completely empty. Can we simplify this and move it to the repository?

Reply:
It would just make it easy to refactor later?

Reply:
It should really be done as part of this PR, to match the other projects.